src/antsibull_docs/data/sphinx_init/requirements_txt.j2

```diff
@@ -4,5 +4,5 @@
 
 antsibull-docs >= 1.0.0, < 2.0.0
 ansible-pygments
-sphinx
+sphinx != 5.2.0  # The != 5.2.0 restriction is thanks due to a bug in the sphinx-rtd-theme. Once this is fixed, this restriction can be removed.
```

Comment:
If the issue is not actually in `sphinx` itself, should we do it like this? presumably a new release of `sphinx` would not fix the issue and we'd be right back where we started.

```suggestion
sphinx < 5.2.0  # The < 5.2.0 restriction is thanks due to a bug in the sphinx-rtd-theme. Once this is fixed, this restriction can be removed.
```

Reply:
Actually the problem isn't 5.2.0, but 5.2.0.post0. I've changed this line to avoid `5.2.0.post0`, which at least fixes the issue in CI for this repo.

Reply:
ah yup, I just found that myself, was about to say the same. Perfect, thanks!